src/main/resources/application.yaml

```diff
@@ -355,14 +355,155 @@ darts:
       common-config:
         thread-pool-size: 5
         system-user-email: ${SYSTEM_USER_EMAIL:}
-      arm-rpo-duration: ${ARM_RPO_DURATION:1h}
-      inbound-annotation-transcription-deleter:
-        default-batch-size: 100000
+      apply-retention:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
```

Comment:
I take it these will all be updated as part of [DMP-4219](https://tools.hmcts.net/jira/browse/DMP-4219)?

Reply:
Yes :) We can probably amend that ticket to just indicate that this config needs updating, this way anytime they grab the users identity the correct user is grabbed (Same for last modified by)